CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+## 2.2.0
+
+### Features and improvements
+* Add `testMode` Map option that silences errors and warnings generated due to an invalid accessToken. It maybe useful when using the library to write unit tests. ([#10445](https://github.com/mapbox/mapbox-gl-js/pull/10445))
```

Comment:
How about?

```suggestion
* Add `testMode` Map option that allows writing of automated tests without an access token. ([#10445](https://github.com/mapbox/mapbox-gl-js/pull/10445))
```

Reply:
I think the map option documentation is slightly more specific about what it actually does (silences errors) and hints at what the limitations are. It might be good to be even more clear about what this supports and what this doesn't support. Are there longer docs anywhere I can pull from?